lib/views/admin/status.js

```diff
@@ -3,19 +3,24 @@ var pug = require('pug')
 
 const os = require('os')
 const config = require('../../config')
+const extend = require('xtend')
 
 module.exports = function (req, res) {
   var locals = {
     config: config.get(),
-    section: 'admin',
-    adminSection: 'status',
-    user: req.user,
     nodeVersion: process.version,
     architecture: os.arch(),
     platform: os.type(),
-    osRelease: os.release(),
-    config: config.get()
+    osRelease: os.release()
+  }
+  if (!req.accepts('text/html')) {
+    return res.status(200).header('content-type', 'application/json').send(JSON.stringify(locals))
```

Comment:
`locals` has config

Reply:
Yep, this is the full config.  I've reduced it to just the config elements that appear in status plus a few more which are not currently on that page, but nothing private.